azure-mgmt-servicebus/src/main/java/com/microsoft/azure/management/servicebus/implementation/OperationsInner.java

```diff
@@ -80,7 +80,7 @@ public OperationsInner(Retrofit retrofit, ServiceBusManagementClientImpl client)
             @Override
             public Page<OperationInner> nextPage(String nextPageLink) {
                 // Temporary work-around for https://github.com/Azure/azure-sdk-for-java/issues/1641
-                if (nextPageLink == "") {
+                if (nextPageLink.isEmpty()) {
```

Comment:
@pgbhagat your PR failed to pass the Travis CI mock testing and the failing test seems to be one in the service bus service. Can you please make a change to the code above and see if the Travis CI is passing:
`if (nextPageLink != null && nextPageLink.isEmpty()) {`

Reply:
Correctly pointed. Added a null check in my new PR.

Reply:
[P3]: Behavior of `nextPageLink == ""` and `nextPageLink.isEmpty()` are same. Agree that for readability isEmpty() is better.